plugins/Rockets/RocketsPlugin.cpp

```diff
@@ -294,16 +286,20 @@ class RocketsPlugin::Impl : public ActionInterface
                          const std::function<PropertyMap(PropertyMap)>& action)
     {
         _bindEndpoint(desc.methodName, [
-            name = desc.methodName, input, action, &engine = _engine
+            name = desc.methodName, action, &engine = _engine
         ](const auto& request) {
-            PropertyMap params = input;
-            if (::from_json(params, request.message))
+            engine.triggerRender();
```

Comment:
Why do you need this? It doesn't look right to trigger rendering from any possible request.

Reply:
maybe not. I have to check